src/makeRobot.js

```diff
@@ -39,6 +39,63 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    goForward(distance = 1) {
+      if (distance < 0) {
+        return this;
+      }
+      this.coords.y += distance;
+
+      return this;
+    },
+    goBack(distance = 1) {
+      if (distance < 0) {
+        return this;
+      }
+      this.coords.y -= distance;
+
+      return this;
+    },
+    goRight(distance = 1) {
+      if (distance < 0) {
```

Comment:
```suggestion
    goBack(distance = 1) {
      if (distance < 0) {
        return this;
      }
      this.coords.y -= distance;

      return this;
    },

    goRight(distance = 1) {
      if (distance < 0) {
```


add blank line between methods, fix everywhere